src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -1438,6 +1439,107 @@ private void WriteRewriter()
+                        {
+                            Blank();
+                            WriteLine(GetVisitFunctionDeclaration(node.Name, isOverride: true));
+                            Brace();
+                            bool hadField = false;
+                            foreach (Field field in AllNodeOrNodeListFields(node))
+                            {
+                                hadField = true;
+                                if (SkipInVisitor(field))
+                                {
+                                    WriteLine($"{field.Type} {ToCamelCase(field.Name)} = node.{field.Name};");
+                                }
+                                else
+                                {
+                                    WriteLine($"{field.Type} {ToCamelCase(field.Name)} = ({field.Type})this.Visit{(IsNodeList(field.Type) ? "List" : "")}(node.{field.Name});");
+                                }
+                            }
+                            if (hadField)
+                            {
+                                WriteLine($"{node.Name} updatedNode;");
+                                Blank();
+                                WriteNullabilityCheck(inverted: false);
+                                Brace();
+                                WriteTypeSymbolRetreival();
+                                WriteUpdate(decl: false, updatedType: true);
+                                WriteNullabilityUpdate();
+                                Unbrace();
+                                WriteLine("else");
+                                Brace();
+                                WriteUpdate(decl: false, updatedType: false);
+                                Unbrace();
+                                WriteLine("return updatedNode;");
+                            }
+                            else
+                            {
+                                WriteNullabilityCheck(inverted: true);
+                                Brace();
+                                WriteLine("return node;");
+                                Unbrace();
+                                Blank();
+                                WriteTypeSymbolRetreival();
+                                WriteUpdate(decl: true, updatedType: true);
+                                WriteNullabilityUpdate();
+                                WriteLine("return updatedNode;");
+                            }
+                            Unbrace();
+
+                            void WriteNullabilityCheck(bool inverted) =>
+                                WriteLine($"if ({(inverted ? "!" : "")}{topLevelNullabilities}.ContainsKey(node))");
+
+                            void WriteTypeSymbolRetreival() =>
+                                WriteLine($"TypeSymbol type = {topLevelNullabilities}[node].TypeSymbol;");
+
+                            void WriteUpdate(bool decl, bool updatedType)
```

Comment:
>WriteUpdate [](start = 33, length = 11)

nit: local functions use camelCase convention

Reply:
Have to add our .editorconfig here.

---
In reply to: [251674760](https://github.com/dotnet/roslyn/pull/32899#discussion_r251674760) [](ancestors = 251674760)